docs-ref-conceptual/includes/cli-install-linux-dnf.md

```diff
@@ -96,8 +100,7 @@ $ sudo make install
 Finally, follow step 1 and 2 of the [install instruction](#install) to add Azure CLI repository. You can now download the package and install it without dependency.
 
 ```bash
-$ sudo yum install yum-utils -y
-$ sudo yumdownloader azure-cli
+$ sudo dnf download azure-cli
```

Comment:
I tried on centos 8, this command cannot be found. The command I used is `dnf install --downloadonly azure-cli --destdir .`.